apps/judicial-system/backend/src/app/modules/case/limitedAccessCase.service.ts

```diff
@@ -445,7 +450,21 @@ export class LimitedAccessCaseService {
               )
             }
 
-            throw new NotFoundException('Defender not found')
+            return this.civilClaimantService
```

Comment:
I feel like the code is getting a little confusing to read, couldn't we refactor it to use await instead of chaining .then()? Just a thought, not a pressing matter.

Reply:
I happen to be a big fan of chaining 😎 but I understand not everyone is. I will take that into account in the future.